src/System.Private.ServiceModel/src/System/ServiceModel/Security/SecurityUtils.cs

```diff
@@ -578,6 +660,20 @@ private static NetworkCredential UnsafeGetDefaultNetworkCredentials()
             {
                 return CredentialCache.DefaultNetworkCredentials;
             }
+
+            internal static string GetCurrentUserIdAsString(NetworkCredential credential)
+            {
+#if FEATURE_NETNATIVE
+                // There's no way to retrieve the current logged in user Id in UWP apps
+                // so returning a username which is very unlikely to be a real username;
+                return "_______****currentUser****_______";
```

Comment:
Can we use something along the lines of this? 
https://code.msdn.microsoft.com/windowsapps/User-domain-name-sample-85ce3e49/sourcecode?fileId=51014&pathId=498305094

(Was using this in a toy app of mine, wondering if we can work it into here)